docs/tutorials/classification.ipynb

```diff
@@ -2,195 +2,465 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "source": [
-    "# Classification\n",
-    "\n",
-    "This tutorial uses safeds on **titanic passenger data** to predict who will survive and who will not, using sex as a feature for the prediction.\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "In this tutorial, we use `safeds` on **Titanic passenger data** to predict who will survive and who will not."
+   ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "1. Load your data into a `Table`, the data is available under `docs/tutorials/data/titanic.csv`:\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "### Load Your Data into a `Table`\n",
+    "The data is available under [Titanic - Machine Learning from Disaster](https://www.kaggle.com/c/titanic/data):\n"
```

Comment:
I should've been more explicit, sorry. The version of the dataset that is used here is from https://github.com/Safe-DS/Datasets/blob/main/src/safeds_datasets/tabular/_titanic/data/titanic.csv.